blue-green-deploy/README.md

```diff
@@ -0,0 +1,86 @@
+# Blue Green Deploy on Kubernetes
+
+THe following task can help you to deploy an application using the Blue-Green deployment strategy. This task gives you the flexibility to deploy a version of the application from the blue zone to the green zone without disturbing the previous version in the blue zone. It works by making the service point to the newer version of the application deployed. 
+
+- If you are deploying the first version of the application then it will go into the blue zone and for that you need to provide deployment and service manifests via the `workspaces` and in the `params` the service name and version. Sample Kubernetes manifests for version v1 can be found [here](./v1-deploy).
+- For further deployments the deployments will get deployed in the other zone to which the current service is pointing to. For example if we have a deployment running in the blue zone then we will deploy the next deployment in the green zone and make the service point to the green zone or vice versa.
+
+## Installing the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/blue-green-deploy/blue-green-deploy.yaml
+```
+
+## Installing the ClusterRoleBinding
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/blue-green-deploy/clusterrolebinding.yaml
+```
+
+## Workspaces
+
+- **manifest-dir**: Manifest files can be provided via the workspaces.(Default: _emptyDir:{}_ in case no manifest is provided)
+- **kubeconfig-dir**: If you want to deploy you application to another cluster then you can mount your `kubeconfig` file via this `workspace`. (Default: _emptyDir:{}_ in case `kubeconfig` is not mounted)
+
+## Parameters
+
+- **SERVICE_NAME**: The service name pointing to the existing deployment. (_Note_: The service name for the new deployment should be same)
+- **NEW_VERSION**: The version of the deployment to be deployed in the green/blue zone
+- **MANIFEST**: The deployment manifest URL file path provided in case the manifest is present on Github. (_Example_: "https://raw.githubusercontent.com/vinamra28/blue-green-deployment-k8s/master/deployment%2Bservice/blue-deployment.yaml")
```

Comment:
```suggestion
- **MANIFEST**: The deployment manifest URL file path provided in case the manifest is present on Github. (_Example_: "https://raw.githubusercontent.com/tektoncd/catalog/master/blue-green-deploy/example/v1-deploy/blue-deployment.yaml")
```